### DISCUSSION TITLE
Detailed questions RE the "Installation from source wiki"

### DISCUSSION THREAD

**Discussion by seamusdemora** [Arkq/bluez-alsa#696] [Q&A] [answered]
I'm planning to build `bluez-alsa` on a RPi 02W running the "Lite" version of RPi OS 'bookworm'. (REF #695)

I'm in `Step 3` of the [install wiki](https://github.com/arkq/bluez-alsa/wiki/Installation-from-source), and have several questions:  

1\. Does `mpg123` support **also** require `--enable-mp3lame`?  
2\. Will I need the `--enable-cli` to use `bluealsa-cli` tool (I have no GUI) ??   
3\. I want to use _some of_ the `systemd` options. I'll add `--enable-systemd` option, but what about others? I do not want my music to stop playing when I log out (as it does with `pipewire`). Can you make recommendations?

**Comment by borine** [comment, marked as answer]
1. already answered in https://github.com/arkq/bluez-alsa/discussions/695#discussioncomment-9132211 - the application (*any* application) just sends audio to an ALSA PCM device (*any* ALSA PCM device), it does not know or care what methods, codecs, magic tricks or whatever that the device uses to get that audio to the speaker.
2. already answered in https://github.com/arkq/bluez-alsa/discussions/695#discussioncomment-9132211 - for a simple use case like this one, bluealsa "just works"
3. My personal advice is to keep it simple - run the bluealsa daemon as root using systemd, which happens by default with "--enable-systemd". 

[EDIT] just a thought - you know that you must arrange for mpg123 to keep running when you log out? That is not something that bluealsa (nor pipewire) can do for you.

**Reply by seamusdemora** [reply]
@borine 

> [EDIT] just a thought - you know that you must arrange for mpg123 to keep running when you log out? That is not something that bluealsa (nor pipewire) can do for you.

I think I've got at least that much covered. :)  
```
nohup mpg123 --loop -1 rainstorm.mp3 &
``` 

Thank you so much for your help with this.
